src/lib/network/edit_nic_name.rb

```diff
@@ -151,5 +162,36 @@ def CheckUdevNicName(name)
 
       true
     end
+
+    # When an interface name has changed, it returns whether the user wants to
+    # update the interface name in the related routes or not.
+    #
+    # return [Boolean] whether the routes have to be updated or not
+    def update_routes?(previous_name)
```

Comment:
Could this also be moved into `Routing` module? Seems to be its responsibility, isn't it?

Reply:
Probably yes, I didn't moved this method and also the update_routing_devices! because avoided to Import "LanItems"